lib/perl/Genome/VariantReporting/Framework/Component/Base.pm

```diff
@@ -50,7 +50,16 @@ sub properties_in_plan {
             implied_by => undef,
             is_structural => undef)) {
         next if $property->{is_param} && $property->property_name =~ m/_md5$/;
-        next if $property->{is_param} && $property->property_name =~ m/_count$/;
+        if ($property->{is_param} && $property->property_name =~ m/^(.*)_count$/) {
```

Comment:
Would it make sense for the `_md5` property to to also check for its related `_count` property?